docs/pages/fund-apps/estimate-fees.mdx

```diff
@@ -0,0 +1,128 @@
+
+- App messages (text, reactions, replies)
+- Media attachments (charged by size, capped at 1 MB)
+- System messages (read receipts, typing indicators)
+
+To learn more, see [Envelope types](/protocol/envelope-types).
+
+Collected messaging fees are paid directly to the node operators who run the globally distributed nodes that power the XMTP Broadcast Network.
+
+#### Messaging fee payment flow
+
+This sequence diagram illustrates the messaging fee payment flow from payers to independent node operators using a series of smart contracts.
+
+<Zoom>
+<Mermaid
+  id="payer-to-node"
+  chart={`
+    sequenceDiagram
+        actor Payer
+        participant PayerRegistry
+        participant PayerReportManager
+        participant DistributionManager
+        participant NodeRegistry
+        actor NodeOperator as Node operator
+
+        Payer->>PayerRegistry: 1. Allocates USDC to cover usage fees
+        Note over PayerReportManager: Node operator submits payer usage reports
+        PayerReportManager->>PayerRegistry: 2. Deducts fees from payer's balance
+        PayerRegistry->>DistributionManager: 3. Transfers collected fees
+        NodeOperator->>DistributionManager: 4. Claims share of fees
+        DistributionManager->>NodeRegistry: 5. Verifies claimant is node owner
+        NodeRegistry-->>DistributionManager: Returns node owner's address
+        DistributionManager->>NodeOperator: 6. Transfers earned fees to node operator
+`}
+/>
+</Zoom>
+
+### Gas fees
+
+Commit messages sent through the **XMTP Appchain** incur **gas fees**. Commit messages require transactions on the XMTP Appchain. Gas fees are paid directly from the payer wallet's XMTP Appchain allowance.
+
+Every commit message an app or agent sends through the XMTP Appchain counts toward its message volume. Commit messages are typically used to perform:
+
+- Group management updated (member additions, permission changes)
+- Identity updates
+- Consent preferences updates
+- Payer-related updates
+
+To learn more, see [Envelope types](/protocol/envelope-types).
+
+Collected gas fees are paid directly to the XMTP Appchain. Fees are used to maintain the Appchain.
+
+## Estimate XMTP fees
+
+Use the [XMTP fee calculator](#TODO) to help you estimate fees to be incurred by your app or agent. This calculator uses network averages, which provides estimates at the correct order of magnitude.
+
+Here are rough estimates of XMTP Broadcast Network and XMTP Appchain fees:
+
+- **Messaging fee**: At launch, a 1 KB text message sent to the XMTP Broadcast Network incurs a fee of ~0.001 USDC (≈ $1 per 1,000 small messages).
+
+- **Gas fee**: A message that triggers a commit to the XMTP Appchain incurs a ≤ 500 gwei fee, which works out to fractions of a cent.
+
+### Pricing and network scale
+
+The network is designed to be profitable at 1B messages/month. As volume grows beyond this point, surplus revenue is recycled to lower the global per-byte price for everyone.
```

Comment:
This kinda sounds like it's never profitable. Which is fine, but we should say so.

Reply:
"profitable" -> "break-even"